src/docs/product/relay/getting-started.mdx

```diff
@@ -187,66 +204,37 @@ docker run --rm -it                \
 ./relay run --config /etc/relay/
 ```
 
-## Logging
-
-Once Relay is running, you will receive `INFO` messages, such as:
-
-```
-INFO  relay::setup > launching relay from config folder .relay
-INFO  relay::setup >   relay mode: managed
-INFO  relay::setup >   relay id: f2119bc9-9a9b-4531-826b-24e9794902f2
-INFO  relay::setup >   log level: INFO
-```
-
-This example displays the default logging level, which you can modify so it displays either more or less information. For details about configuring logging please see [Logging](/product/relay/options/#logging) on the options page.
-
-## Health Checks
-
-Relay provides two URLs for checking system status:
-
-- `GET /api/relay/healthcheck/live/`: Tests if Relay is running and listening to
-  HTTP requests.
-- `GET /api/relay/healthcheck/ready/`: Tests if Relay is authenticated with the
-  upstream and operating normally.
-
-In case of success, both endpoints return a _200 OK_ response:
-
-```json
-{
-  "is_healthy": true
-}
-```
-
 ## Sending a Test Event
 
-Once Relay is running and authenticated with Sentry, send a test event.
+Once Relay is running and authenticated with Sentry, send a test event to one of the projects in your organization.
 
-Get the DSN of your project by navigating to _Project Settings > Client Keys
-(DSN)_, which looks similar to:
+Get the DSN of your project by navigating to your project settings ( _Settings > Projects > {YOUR_PROJECT_NAME} > Client Keys (DSN)_ ), and selecting one of the existing DSN value, which looks similar to:
 
-```
-https://12345abcdb1e4c123490ecec89c1f199@o1.ingest.sentry.io/2244
-```
+`https://12345abcdef10111213141516171819@o1.ingest.sentry.io/2345`
 
 Next, replace parts of the DSN to match the address at which Relay is reachable.
 For instance, if Relay listens at `http://localhost:3000`, change the protocol
 and host of the DSN to:
 
-```
-http://12345abcdb1e4c123490ecec89c1f199@localhost:3000/2244
-```
+<code>
+<strong><i>http</i></strong>://12345abcdef10111213141516171819@<strong><i>localhost:3000</i></strong>/2345
+</code>
 
 Use the new DSN in your SDK configuration. To test this, you can send a message
-with `sentry-cli`:
+with `sentry-cli` ([installation instructions](/cli/installation/)):
 
 ```shell
-export SENTRY_DSN='http://12345abcdb1e4c123490ecec89c1f199@127.0.0.1:3000/2244'
+export SENTRY_DSN='http://12345abcdef10111213141516171819@127.0.0.1:3000/2345'
 sentry-cli send-event -m 'A test event'
 ```
 
 In a few seconds, the event should appear in the issues stream in your
 project.
 
-## Advanced Configuration
+
+## Next Steps
+
+You have a functioning Relay instance, great! Make sure to check [Operating Guidelines](/product/relay/best-practices/) to learn how to monitor and scale your Relay setup.
```

Comment:
While I really like this, the tone of this document is generally very dry and neutral. I'd suggest:

```suggestion
You have a working Relay instance. Make sure to check [Operating Guidelines](/product/relay/best-practices/) to learn how to monitor and scale your Relay setup.
```